src/app/modules/main/chat_section/module.nim

```diff
@@ -280,24 +281,32 @@ proc rebuildCommunityTokenPermissionsModel(self: Module) =
 
   let community = self.controller.getMyCommunity()
   var tokenPermissionsItems: seq[TokenPermissionItem] = @[]
-  var allTokenRequirementsMet = false
 
   for id, tokenPermission in community.tokenPermissions:
-    # TODO: for startes we only deal with "become member" permissions
-    if tokenPermission.`type` == TokenPermissionType.BecomeMember:
-      let tokenPermissionItem = self.buildTokenPermissionItem(tokenPermission)
+    let tokenPermissionItem = self.buildTokenPermissionItem(tokenPermission)
+    tokenPermissionsItems.add(tokenPermissionItem)
+
+  let memberPermissions = filter(tokenPermissionsItems, tokenPermissionsItem => 
```

Comment:
minor: 
It seems that this is more of `memberRequirements` and `memberRequirementsMet` (instead of `memberPermissions` and `memberPermissionsMet`). 
You also use `tokenRequirementsMet` later when checking those properties.

It just confused me a bit, but keep in mind that I'm new to this topic, so maybe I just didn't get it right 🙂

Reply:
I'm fine with how it is, but changing to `memberRequirements` is fine too.